include/mbedtls/ssl.h

```diff
@@ -107,6 +107,8 @@
 #define MBEDTLS_ERR_SSL_TIMEOUT                           -0x6800  /**< The operation timed out. */
 #define MBEDTLS_ERR_SSL_CLIENT_RECONNECT                  -0x6780  /**< The client initiated a reconnect from the same port. */
 #define MBEDTLS_ERR_SSL_UNEXPECTED_RECORD                 -0x6700  /**< Record header looks valid but is not expected. */
+#define MBEDTLS_ERR_SSL_NON_FATAL                         -0x6680  /**< The message is a non-fatal error. */
```

Comment:
I think I mentioned this before: the name of this macro `MBEDTLS_ERR_SSL_NON_FATAL` is to broad and does not hint much about the actual problem.